Update pt_BR.po

marcelocripe opened the pull request:
Hello realh.

Please accept the update of the "pt_BR.po" file that contains revisions and improvements to the translations of the “pt_BR” language.

Thank you very much for programming, developing and keeping ROXTerm alive.

marcelocripe
(Original text in Brazilian Portuguese language)

- - - - -

Olá, realh.

Por favor, aceite a atualização do arquivo "pt_BR.po" que possui as revisões e melhorias das traduções do idioma “pt_BR”.

Muito obrigado por programar, desenvolver e manter o ROXTerm vivo.

marcelocripe
(Texto original em idioma Português do Brasil)

marcelocripe committed:
Update pt_BR.po

marcelocripe commented:
Hello realh.

> Thanks for this.

I'm the one who thanks you for programming, developing and keeping ROXTerm alive, and still accepting my "pt_BR" translations.


> Would you be able to translate the XHTML and man pages (the source is Docbook XML) by simply editing copies of the XML files?

Yes, I can translate files in XHTML or XML format and manual pages, as long as I know exactly what I can and cannot translate. I have an example of an XML file that I translated to "pt_BR" and it is on the page https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix, to be more specific it is the file https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix/-/blob/main/translation_pt_BR.xml?ref_type=heads.

In this example, the original language is Russian and there was an English translation https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix/-/blob/main/translation.xml?ref_type=heads that was made by the creator of Phoenix. So it was easy for me to recognize the place where I would need to translate. For example, in the <translation>Edit Information</translation> entries, I edited it to <translation>Editar as informações</translation>. I was able to test the "pt_BR" translation and I make it available on my GitLab page, since Altmer seems to no longer be maintaining the development and updates of Phoenix.


> If you would prefer to work entirely with pot/po files I should be able to add support for that.

Without a doubt, the ".pot" and ".po" files are easier for most volunteer translators, because they can use Poedit or a simple text editor to translate them. Furthermore, the ".pot" and ".po" files can be submitted to Transifex.

I don't know if these ".po" files (for_use_antix-contribs_roxterm_2025-09-18.zip) from the old version are still useful to you. Unfortunately, few volunteer translators volunteered their time to translate the ROXTerm that I submitted to Transifex (https://app.transifex.com/antix-linux-community-contributions/antix-contribs/roxterm/) on 04-09-2024. Some languages ​​were fully translated in the old version of ROXTerm (before its most recent update on 11-09-2025). It was the volunteer Hungarian translator who notified the antiX forum that you had updated the project. Without this information, I wouldn't have known that ROXTerm had been updated. My email address is public. If you'd like to notify me by email whenever ROXTerm is updated, this will help me keep the program updated with the latest "pt_BR" translation.

- - - - -

Olá, realh.

> Thanks for this.

Sou eu que agradeço por você programar, desenvolver e manter o ROXTerm vivo, e ainda aceitar as minha traduções "pt_BR".


> Would you be able to translate the XHTML and man pages (the source is Docbook XML) by simply editing copies of the XML files?

Sim, eu consigo traduzir os arquivos no formato XHTML ou XML e as páginas do manual, desde que eu saiba exatamente o que posso traduzir e o que não posso traduzir. Eu tenho um exemplo de um arquivo XML que traduzi para "pt_BR" e está na página https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix, para ser mais específico é o arquivo https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix/-/blob/main/translation_pt_BR.xml?ref_type=heads.

Neste exemplo, o idioma original é o russo e havia uma tradução para o idioma inglês https://gitlab.com/marcelocripe/emulador_de_jogos_phoenix/-/blob/main/translation.xml?ref_type=heads que foi feita pelo criador do Phoenix. Então ficou fácil para eu conseguir reconhecer o local onde eu teria que traduzir, Por exemplo, nas entradas <translation>Edit Information</translation> eu editei para <translation>Editar as informações</translation>. Eu consegui testar a tradução "pt_BR" e a disponibilizo na minha página do GitLab, já que o Altmer parece que não está mais mantendo o desenvolvimento e a atualização do Phoenix.


> If you would prefer to work entirely with pot/po files I should be able to add support for that.

Sem dúvida alguma, os arquivos ".pot" e ".po" são mais fáceis para a maioria dos tradutores voluntários, porque podem utilizar o Poedit ou um editor de texto simples para traduzirem, e ainda, os arquivos ".pot" e ".po" podem ser enviados para o Transifex.

Eu não sei se estes arquivos ".po" (for_use_antix-contribs_roxterm_2025-09-18.zip) da versão antiga ainda são úteis para você. Infelizmente, poucos tradutores voluntários doaram o seu tempo para traduzir o ROXTerm que eu enviei para o Transifex (https://app.transifex.com/antix-linux-community-contributions/antix-contribs/roxterm/) no dia 04-09-2024. Alguns idiomas foram totalmente traduzidos na versão antiga do ROXTerm (antes da sua atualização mais recente do dia 11-09-2025). Foi o tradutor voluntário do idioma húngaro que avisou no fórum do antiX que você havia atualizado o projeto, sem esta informação eu não teria como saber que o ROXTerm foi atualizado. O meu endereço de e-mail é público, se você quiser e puder me avisar por e-mail sempre que o ROXTerm for atualizado, isso me ajudará a manter o programa com a tradução mais recente do idioma "pt_BR".

[for_use_antix-contribs_roxterm_2025-09-18.zip](https://github.com/user-attachments/files/22411748/for_use_antix-contribs_roxterm_2025-09-18.zip)

marcelocripe commented:
realh, I'm sorry, but I can't convert files using the summarized instructions. I can only create or convert files when the step-by-step instructions are quite detailed. Creating a ".po" file from a ".pot" file and creating a ".mo" file from a ".po" file were taught to me on the antiX forum.

Which of the two files should I translate?

https://github.com/realh/roxterm/blob/master/docs/pt_BR/index.html
and/or
https://github.com/realh/roxterm/blob/master/po/xhtml/xhtml.pot

I can translate the "index.html" file directly and/or the "xhtml.pot" file.

Or, if you prefer, you can email me the files and I'll translate them in whatever format they're in. If there's another file in another language, I'll let you know which locations can and can't be translated.

However, I will only be able to return to translating for ROXTerm once I have completed the translations of the resources https://app.transifex.com/anticapitalista/antix-development/antix23-desktop-files/ (with 2,190 entries to be translated) and https://app.transifex.com/anticapitalista/antix-development/antix23-desktop-files-extra/ (with 35 entries to be translated). These ".desktop" files will be used in antiX 25 (the next stable release), which will be released sometime in 2025, and will also be used for antiX 23 (the current stable release).

- - - - -

realh, eu sinto muito, mas não consigo fazer as conversões com as instruções resumidas. Eu só consigo fazer a criação ou conversão dos arquivos quando o passo a passo está bastante detalhado. Criar um arquivo ".po" a partir de um arquivo ".pot" e criar um arquivo ".mo" a partir de um arquivo ".po" me ensinaram no fórum do antiX.

Qual dos dois arquivos eu tenho que traduzir?

https://github.com/realh/roxterm/blob/master/docs/pt_BR/index.html
e/ou
https://github.com/realh/roxterm/blob/master/po/xhtml/xhtml.pot

Eu posso traduzir diretamente o arquivo "index.html" e/ou no "xhtml.pot".

Ou se você preferir, pode me enviar os arquivos por e-mail e o traduzirei no formato que estiver. Se houver um outro arquivo de um outro idioma, eu saberei os locais que podem ser traduzidos e os locais onde não podem ser traduzidos.

Contudo, eu só poderei voltar a traduzir para o ROXTerm quando eu terminar as traduções dos recursos https://app.transifex.com/anticapitalista/antix-development/antix23-desktop-files/ (com 2190 entradas para serem traduzidas) e do https://app.transifex.com/anticapitalista/antix-development/antix23-desktop-files-extra/ (com 35 entradas para serem traduzidas). Estes arquivos ".desktop" serão utilizados no antiX 25 (a próxima edição estável) que será lançado em algum momento ainda neste ano de 2025, e também servirão para o antiX 23 (a atual edição estável).

marcelocripe commented:
Hello, [realh](https://github.com/realh).

With your detailed instructions, I was able to complete the procedure in antiX 23. I'm not sure if everything I did is correct. Could you please check if I did everything correctly?

The commands I applied are in the files "ROXTerm Help Translation.txt" and "Traducao da Ajuda do ROXTerm.txt" in the ".zip" archive.

As I mentioned before, I'll only be able to dedicate my limited free time to ROXTerm again after finishing the text entries in the ".desktop" files of the "antix23-desktop-files" resource (I translated entry 216 of 2190) and "antix23-desktop-files-extra" (with 35 entries to be translated). Transifex's artificial intelligence is hindering the volunteer translation work (https://www.antixforum.com/forums/topic/antix23-desktop-files-resource-at-transifex-has-been-updated/). I ask that you be patient, because the texts in the ".desktop" files directly influence the quality of the menus translated into my language "pt_BR", and also offer a much higher quality for the understanding of Brazilian end users when handling antiX Linux for the first time.

- - - - -

Olá, [realh](https://github.com/realh).

Com as suas instruções detalhadas eu consegui realizar o procedimento no antiX 23. Eu não sei se tudo que fiz está correto, por favor, você poderia verificar se fiz tudo corretamente?

Os comandos que apliquei estão nos arquivos "ROXTerm Help Translation.txt" e "Traducao da Ajuda do ROXTerm.txt" do arquivo ".zip".

Como eu havia dito antes, só poderei dedicar o meu pouco tempo livre no ROXTerm novamente após finalizar os textos das entradas dos arquivos ".desktop" dos recursos "antix23-desktop-files" (traduzi a entrada 216 de 2190) e do "antix23-desktop-files-extra" (com 35 entradas para serem traduzidas), a inteligência artificial do Transifex está dificultando do trabalho voluntário de tradução (https://www.antixforum.com/forums/topic/antix23-desktop-files-resource-at-transifex-has-been-updated/). Eu peço que você tenha paciência, porque os textos dos arquivos ".desktop" influenciam diretamente na qualidade dos menus traduzidos em meu idioma "pt_BR", e ainda, oferecem uma qualidade muito maior para a compreensão dos usuários finais brasileiros ao manusearem o antiX Linux pela primeira vez.

[Traducao da Ajuda do ROXTerm.zip](https://github.com/user-attachments/files/22480639/Traducao.da.Ajuda.do.ROXTerm.zip)

realh commented:
The part about GTK  < 3.10 is obsolete, so I've removed two of the paragraphs from the  "Keyboard Shortcuts" section and modified the remaining paragraph. I changed your translation manually, with a little help from Google Translate. Please could you check that this paragraph is still good Portuguese?

> O ROXTerm permite que os perfis dos atalhos podem ser editados em um editor de texto, seja por meio do Gerenciador das Configurações ou por menu de um emulador terminal (no submenu das ‘Preferências’). Os nomes utilizados nas ações correspondem aos rótulos localizados nos itens do menu. Os novos atalhos devem ser recarregados automaticamente ao salvar o arquivo. Opcionalmente, você pode selecionar um esquema no menu das ‘Preferências’ que agora força o recarregamento.

marcelocripe commented:
Hello realh.

> The part about GTK < 3.10 is obsolete, so I've removed two of the paragraphs
> from the "Keyboard Shortcuts" section and modified the remaining
 > paragraph. I changed your translation manually, with a little help from
> Google Translate. Please could you check that this paragraph is still good
> Portuguese?

I just read your reply and have already made the necessary adjustments to the new text in the "index.html" file. I'll send you the file in the merge request https://github.com/realh/roxterm/pull/279. Since you didn't send me the new text here in this thread, I had to copy the new text from the "en" language file and paste it into Google and DeepL automatic translators. After understanding the context of the automatic translation, I'll make the necessary adjustments to make it easier for Brazilians to understand.

Thank you very much for checking the text I suggested and making the changes you deemed necessary.

The ROXTerm help text looks great in a web browser.

- - - - -

Olá, realh.

> The part about GTK < 3.10 is obsolete, so I've removed two of the paragraphs
> from the "Keyboard Shortcuts" section and modified the remaining
 > paragraph. I changed your translation manually, with a little help from
> Google Translate. Please could you check that this paragraph is still good
> Portuguese?

Eu acabei de ler a sua resposta e já fiz os ajustes necessários no novo texto do arquivo "index.html" e já vou te enviar o arquivo no pedido de mesclagem https://github.com/realh/roxterm/pull/279. Como você não me enviou o novo texto aqui nesta conversa, eu precisei copiar o novo texto no arquivo do idioma "en" e colar no tradutor automático do Google e do DeepL. Após eu compreender o contexto da tradução automática, eu faço os ajustes que julgo necessário para facilitar a compreensão dos brasileiros.

Muito obrigado por verificar o texto que indiquei e ainda fazer as alterações que julgou necessário.

O texto da ajuda do ROXTerm ficou muito bonito de ser visto no navegador de internet.
